tests/ixia/test_pfc_pause_lossless.py

```diff
@@ -0,0 +1,205 @@
+                                    ip_start=vlan_ip_addrs[0],
+                                    ip_incr_step='0.0.0.1',
+                                    gw_start=gw_addr, 
+                                    gw_incr_step='0.0.0.0')
+
+    topo_sender = create_topology(session=session, 
+                                  name="Sender", 
+                                  ports=list(tx_port), 
+                                  ip_start=vlan_ip_addrs[1], 
+                                  ip_incr_step='0.0.0.1', 
+                                  gw_start=gw_addr, 
+                                  gw_incr_step='0.0.0.0')
+
+    start_protocols(session)
+    
+    test_traffic = create_ipv4_traffic(session=session,
+                                       name='Test Data Traffic',
+                                       source=topo_sender,
+                                       destination=topo_receiver,
+                                       pkt_size=DATA_PKT_SIZE,
+                                       duration=exp_dur,
+                                       rate_percent=50,
+                                       start_delay=1,
+                                       dscp_list=test_dscp_list,
+                                       lossless_prio_list=test_prio_list)
+
+    background_traffic = create_ipv4_traffic(session=session,
+                                             name='Background Data Traffic',
+                                             source=topo_sender,
+                                             destination=topo_receiver,
+                                             pkt_size=DATA_PKT_SIZE,
+                                             duration=exp_dur,
+                                             rate_percent=50,
+                                             start_delay=1,
+                                             dscp_list=bg_dscp_list,
+                                             lossless_prio_list=None)
+    
+    # Pause time duration (in second) for each PFC pause frame.
+    pause_dur_per_pkt = 65535 * 64 * 8.0 / (port_bw * 1000000) 
+    
+    # Do not specify duration here as we want it keep running.
+    pfc_traffic = create_pause_traffic(session=session,
+                                       name='PFC Pause Storm',
+                                       source=rx_port,
+                                       pkt_per_sec=1.1/pause_dur_per_pkt,
+                                       start_delay=0,
+                                       global_pause=False,
+                                       pause_prio_list=test_prio_list)
+    
+    start_traffic(session)
+    
+    # Wait for test and background traffic to finish.
+    time.sleep(exp_dur + 1.5)
+    
+    # Capture traffic statistics.
+    flow_statistics = get_traffic_statistics(session)
+    logger.info(flow_statistics)
+        
+    for row_number, flow_stat in enumerate(flow_statistics.Rows):
```

Comment:
Would like to also see the following cases covered:
1. unpausing the pfc flow and verifying that the test traffic and background traffic are unaffected
2. Ensure that pfc stats are increasing for the right priority when pause frames are being sent

Reply:
I think I took care about the second one also (in the enhancement). I stopped the pause frame (P3, P4) --> verified the test traffic (P3, P4)is going + background traffic is going. I started pause frame (P3, P4) --> verified that Test traffic (P3,P4) stopped  + Backgound is going. That should be enough.

Reply:
I think both the checking are there in the script.